docs/mons.md

```diff
@@ -42,102 +42,143 @@ kubectl rook-ceph mons restore-quorum c
 Before the restore proceeds, you will be prompted if you want to continue.
 For example, here is the output in a test cluster up to the point of starting the restore.
 
-```
+```bash
 $ kubectl rook-ceph mons restore-quorum c
-Info: mon c state is leader
-mon=b, endpoint=192.168.64.168:6789
-mon=c, endpoint=192.168.64.167:6789
-mon=a, endpoint=192.168.64.169:6789
-Info: Check for the running toolbox
-Info: Waiting for the pod from deployment "rook-ceph-tools" to be running
-deployment.apps/rook-ceph-tools condition met
-
-Warning: Restoring mon quorum to mon c (192.168.64.167)
-Info: The mons to discard are: b a
-Info: Are you sure you want to restore the quorum to mon "c"? If so, enter: yes-really-restore
+# Info: mon c state is leader
+# mon=b, endpoint=192.168.64.168:6789
```

Comment:
Do all of these lines have the `#` prefix? I thought some output lines don't have it.

Reply:
`#` prefix is only to separate output from commands.  We don't add `#`  in front of the command output.

Reply:
I'd suggest if there are commands in the same block with the output, the command would have the `$` prefix, while the output is exactly as we see it in the console. I thought this was the pattern for the rook docs, which we could follow here as well.

Reply:
I have removed `#` from the output now and slowly going in the future I feel maintaining the long output will become maintenance burden

Reply:
Yeah let's discuss about long term